app/src/main/java/com/example/myrecipes/ui/Constants.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.myrecipes.ui
+
+class Constants {
+}
```

Comment:
You don't need Constants class here, you can remove it and declare constant fields as top level